src/aws-sdk/sdk.ts

```diff
@@ -58,6 +58,55 @@ class ServiceProxy {
   }
 }
 
+/**
+ * Step Functions AWS Service Integrations support integrations with many services, however, it has a list of
+ * services or methods that are not supported.
+ *
+ * https://docs.aws.amazon.com/step-functions/latest/dg/supported-services-awssdk.html
+ */
+const SFN_SERVICE_BLOCK_LIST: {
+  [key in keyof TSDK]?: true | (keyof TSDK[key])[];
+} = {
+  /**
+   * Api Gateway Management API is just a wrapper around the Invoke API.
+   * Use the API Gateway invoke integration instead.
+   * https://github.com/functionless/functionless/issues/443
+   *
+   * note: when lambda is supported by $AWS.SDK block list this API:
+   *       https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/ApiGatewayManagementApi.html#postToConnection-property
+   */
+  ApiGatewayManagementApi: true,
+  Discovery: ["describeExportConfigurations"],
+  CodeDeploy: [
+    "batchGetDeploymentInstances",
+    "getDeploymentInstance",
+    "listDeploymentInstances",
+    "skipWaitTimeForInstanceTermination",
+  ],
+  ComprehendMedical: ["detectEntities"],
+  DirectConnect: [
+    "allocateConnectionOnInterconnect",
+    "describeConnectionLoa",
+    "describeConnectionsOnInterconnect",
+    "describeInterconnectLoa",
+  ],
+  EFS: ["createTags"],
+  ElasticTranscoder: ["testRole"],
+  EMR: ["describeJobFlows"],
+  Iot: [
+    "attachPrincipalPolicy",
+    "listPrincipalPolicies",
+    "detachPrincipalPolicy",
+    "listPolicyPrincipals",
+  ],
+  Lambda: ["invokeAsync"],
```

Comment:
Oh no, this isn't supported? That seems ... so arbitrary ...

Reply:
Its supported via the optimized integrations. Request/Response on a lambda.

> It is also possible to call Lambda asynchronously using the InvocationType parameter, as seen in the following example:

Reply:
Is it possible to polyfill or is the interface botched?